src/components/Search/Results.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import 'react-native'
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react-native'
+
+import mockData from '../../mock-data/results'
+
+import SearchResults from './Results'
+
+describe( '<SearchResults />', () => {
+  it( 'should scroll down to end of the list', () => {
+    const onEndReached = jest.fn()
+
+    const { getByTestId } = render(
+      <SearchResults data={mockData} testID="search-results" onEndReached={onEndReached} />,
+    )
+
+    const eventData = {
+      nativeEvent: {
+        contentOffset: {
+          y: 500,
+        },
+        // Dimensions of the scrollable content
+        contentSize: {
+          height: 500,
+          width: 100,
+        },
+        // Dimensions of the device
+        layoutMeasurement: {
+          height: 100,
+          width: 100,
+        },
+      },
+    }
```

Comment:
stylistically more compact :)
```suggestion
      nativeEvent: {
        contentOffset: { y: 500 },
        contentSize: { height: 500, width: 100 },
        layoutMeasurement: { height: 100, width: 100 },
      },
    }
```

Reply:
Tbh, inline it if you want in place of `eventData` below